packages/components/src/Autocomplete/Autocomplete.tsx

```diff
@@ -16,11 +16,31 @@ interface AutocompleteProps {
    */
   readonly value: Option | undefined;
 
+  /**
+   * Hint text that goes above the value once the form is filled out.
+   */
+  readonly menuSize?: "medium";
+
   /**
    * Hint text that goes above the value once the form is filled out.
    */
   readonly placeholder: string;
 
+  /**
+   * Turn on separators between result elements.
+   */
+  readonly separators?: boolean;
+
+  /**
+   * Turn off checkmarks on selected result elements.
+   */
+  readonly checkmarks?: boolean;
+
+  /**
+   * Turn on browser autocomplete for the input.
+   */
+  readonly browserAutocomplete?: "off" | "";
```

Comment:
even after adding this I still see the browser autocomplete in the dropdown 
![image](https://user-images.githubusercontent.com/49168885/88853213-9c97de00-d1ac-11ea-978e-94ccfb341171.png)

Reply:
Why are these "off" and blank string? Seems like a weird way to describe a boolean...

Reply:
It's actually not a boolean as per the spec. Autocomplete can be se to a string that will match autocomplete types. It's a pretty wacky spec. https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete